src/lib/remark-plugins/rewrite-tutorial-links/index.ts

```diff
@@ -108,19 +116,19 @@ function handleRewriteTutorialsLink(node: Link | Definition) {
 
       // handle rewriting collection and tutorial dev portal paths
       if (isDocsPath) {
-        node.url = handleDocsLink(nodePath, product as ProductSlug)
+        newUrl = handleDocsLink(nodePath, product as ProductSlug)
       } else if (isCollectionPath) {
-        node.url = handleCollectionLink(nodePath)
+        newUrl = handleCollectionLink(nodePath)
       } else if (isTutorialPath) {
-        node.url = handleTutorialLink(nodePath, TUTORIAL_MAP)
+        newUrl = handleTutorialLink(nodePath, tutorialMap)
       } else if (isProductHubPath) {
-        node.url = `${nodePath}/tutorials`
+        newUrl = `${nodePath}/tutorials`
       }
 
-      if (!node.url) {
+      if (!newUrl) {
         // If the link wasn't found in the map, default to original link
         // Could be a typo, its up to the author to correct -- this feedback should help
-        node.url = nodePath
+        newUrl = nodePath
```

Comment:
Hmmm....we definitely want the default behavior to still return the url.  We may want to return the `newUrl` in the catch statement to retain this?

Reply:
☝️ +1 that was my understanding! the stuff in `try` tries to execute, might get `caught`, but regardless of error or not, everything after the `try-catch` is always executed 🤖